src/components/Footer.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { FilterBy } from '../types/FilterBy';
+
+type Props = {
+  todos: Todo[];
+  filterBy: FilterBy;
+  setFilterBy: (value: FilterBy) => void;
+};
+
+export const Footer: React.FC<Props> = ({ todos, filterBy, setFilterBy }) => (
+  <footer className="todoapp__footer">
+    <span className="todo-count">
+      {todos.filter(todo => !todo.completed).length}
+      items left
```

Comment:
can we create a variable for `todos.filter(todo => !todo.completed).length`?

```suggestion
      {todos.filter(todo => !todo.completed).length}{" "}
      items left
```